scripts/emerge/resolve_drs_uris.py

```diff
@@ -0,0 +1,80 @@
+"""This script resolves drs uris to gs paths."""
+
+import argparse
+import json
+from pydoc import resolve
+import requests
+from pprint import pprint
+
+from oauth2client.client import GoogleCredentials
+
+
+def get_access_token():
+    """Get access token."""
+
+    scopes = ["https://www.googleapis.com/auth/userinfo.profile", "https://www.googleapis.com/auth/userinfo.email"]
+    credentials = GoogleCredentials.get_application_default()
+    credentials = credentials.create_scoped(scopes)
+
+    return credentials.get_access_token().access_token
+
+
+def write_list_to_file(input_list, outfile_name):
+    """Write a list of values to a file."""
+
+    with open(outfile_name, "w") as outfile:
+        for item in input_list:
+            outfile.write(f"{item}\n")
+
+
+def resolve_drs_uri(drs_uuid):
+    """Resolve a drs uri to return the corresponding gs path."""
+
+    uri = f"https://data.terra.bio/ga4gh/drs/v1/objects/{drs_uuid}"
+
+    headers = {"Authorization": "Bearer " + get_access_token(), "accept": "*/*"}
+
+    response = requests.get(uri, headers=headers)
+    status_code = response.status_code
+
+    if status_code != 200:
+        return response.text
+
+    # get a list of the available access methods for a drs uri
+    avail_access_methods = response.json()["access_methods"]
+    # if there is an access method with "type" == "gs", get the gs url (other type options = https)
+    gs_url = [access_type["access_url"]["url"] for access_type in avail_access_methods if access_type["type"] == "gs"] # ["gs://"] or []
+
+    # if a gs url is not available, return "NA"
+    if not gs_url:
+        return "NA"
+    
+    # if gs url is available, return gs path
+    return gs_url[0]
```

Comment:
i suggest checking that the length of `gs_url` is 1, and if it's not, at least printing a warning that multiple paths were found. i don't know if this is possible, but it would be good to flag it in case it does happen.

Reply:
yeah makes sense! i think either commenting that we are confident about the assumption that there will only ever be one (once someone from jade confirms), or adding an assertion or check (possibly in either case, but it's really up to you), would be great.